server/views/includes/index-footer.pug

```diff
@@ -0,0 +1,18 @@
+footer#link-to-contacts.section-footer
+        .row
+          .col.footer-items
+            p Contacts
+          .col.footer-items.copyrightstamp
+            p &copy ch-cat.ru #{copyrightYear}
+        .row
+          .col.footer-items
+            a("mailto: chcatblog@gmail.com")
+              i.fas.fa-at
+            a("#")
+              i.fab.fa-facebook
+            a("https://twitter.com/chcatblog")
+              i.fab.fa-twitter
+            a("#")
+              i.fab.fa-vk
+          .col.footer-items.copyrightstamp
+            p All rights reserved
```

Comment:
New line missing at the end - there's a vs code setting which will always add a new line for you if you're missing one - https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files :)

Reply:
Thank you, updated.